spec/user_interface_spec.rb

```diff
@@ -72,4 +72,61 @@
 
     expect(ui.run).to eq(1)
   end
+
+  describe 'ask for fields method' do
```

Comment:
```ruby
describe '#ask_for_fields' do
```

Check RSpec best practices on `describe` blocks here: https://www.betterspecs.org/#describe

Reply:
You added the `#` but then you left spaces, it is supposed to be the name of the method you are testing, as I typed up there, it should be 

```ruby
describe '#ask_for_fields' do
```